tools/requirements.txt

```diff
@@ -0,0 +1 @@
+pyclipper
```

Comment:
Could we maybe also add requests here? It wasn't available in my newly created virtual environment, so as we now have a requirements.txt it would make sens to add it as well imho.